Description: Avoid an uninitialized variable warning in the sa_len type check.
Forwarded: no
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2016-04-18

--- a/osdep.m4
+++ b/osdep.m4
@@ -401,7 +401,9 @@
 #include <sys/socket.h>], [
 struct sockaddr sa;
 $type *sa_len_ptr;
-sa_len_ptr = &sa.sa_len;],
+sa_len_ptr = &sa.sa_len;
+if (((unsigned long long)sa_len_ptr) % 2 != 0)
+  return 0;],
        [AC_DEFINE_UNQUOTED(sa_len_type, [$type], [sa_len type])
         sa_len_type_found=t
         break])
